code/modules/reagents/reagent_containers/drinks.dm

```diff
@@ -176,6 +176,13 @@
 /obj/item/chems/drinks/coffee/populate_reagents()
 	reagents.add_reagent(/decl/material/liquid/drink/coffee, reagents.maximum_volume)
 
+/obj/item/chems/drinks/coffee/nullcaf
+	name = "cup of null"
```

Comment:
I feel like this is going to be confusing for people who don't understand what nullcaf is. I figure this is probably supposed to be a reference to the old `nothing` reagent but it doesn't really work here.